model/src/main/scala/uk/gov/ons/addressIndex/model/config/AddressIndexConfig.scala

```diff
@@ -39,13 +39,17 @@ case class IndexesConfig(
 )
 
 case class DemouiConfig (
-   defaultLanguage: String,
-   apiURL: String
+  customErrorDev: Boolean,
+  customErrorTest: Boolean,
+  customErrorProd: Boolean,
+  apiURL: String
```

Comment:
`api: ApiConfig`

```
case class(
  host: String,
  port: Int
)
```

Change `AddressIndexClientInstance` to interpolate appropriately